### PR TITLE
docs: update release notes for 31 may

### PR DESCRIPTION
**Pull request by raviteja83**

### CODE REVIEW

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 1]
```diff
@@ -6,12 +6,27 @@ description: Release Notes for 100ms JavaScript SDK
...
+Released: `@100mslive/hms-video-store@0.3.0`, `@100mslive/react-sdk@0.1.0`, `@100mslive/hms-video-react@0.4.0`
+
+Fixed:
+
+-   Video freezing when same video track is played in multiple video elements and one of them is removed
+-   Incorrect device showing in device selection when initially joined as muted.
+
+Breaking:
+
+-   All the requests(RoleChange, track mute/unmute etc) will have peer/requestBy as optional now.
```
Comment (raviteja83) [inline docs/javascript/v2/release-notes/release-notes.mdx:26]: how do we make this more generic? will it be understandable?

Comment (triptu) [reply docs/javascript/v2/release-notes/release-notes.mdx:26]: let's add examples?

For Example:
- In the chat message object, message.sender is now optional
- in unmute request, request.requestedBy is optional

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 2]
```diff
@@ -6,12 +6,27 @@ description: Release Notes for 100ms JavaScript SDK
...
+Fixed:
+
+-   Video freezing when same video track is played in multiple video elements and one of them is removed
+-   Incorrect device showing in device selection when initially joined as muted and then unmuted.
+
+Breaking:
+
+-   All the requests(RoleChange, track mute/unmute etc) will have peer/requestBy as optional now.
+
+> NOTE: if you are planning to use api's to send chat messages, the sdk version should be minimun at this version.
```
Comment (triptu) [inline docs/javascript/v2/release-notes/release-notes.mdx:28]: "send chat messages, update metadata or change roles"

Comment (raviteja83) [reply docs/javascript/v2/release-notes/release-notes.mdx:28]: are apis already available for all of them?

**Review by triptu — APPROVED** [review]

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 3]
```diff
@@ -6,12 +6,30 @@ description: Release Notes for 100ms JavaScript SDK
...
+Fixed:
+
+-   Video freezing when same video track is played in multiple video elements and one of them is removed
+-   Incorrect device showing in device selection when initially joined as muted and then unmuted.
+
+Breaking:
+
+-   All the requests will have the sender as optional now.
+    Examples:
+    -   In chat message object, message.sender could be optional
```
Comment (triptu) [inline docs/javascript/v2/release-notes/release-notes.mdx:28]: could be optional => is optional

**Pull request merged**